internal/handlers/handler_firstfactor.go

```diff
@@ -104,7 +105,8 @@ func FirstFactorPost(msInitialDelay time.Duration, delayEnabled bool) middleware
 
 		if !userPasswordOk {
 			ctx.Logger.Debugf("Mark authentication attempt made by user %s", bodyJSON.Username)
-			ctx.Providers.Regulator.Mark(bodyJSON.Username, false) //nolint:errcheck // TODO: Legacy code, consider refactoring time permitting.
+
+			_ = ctx.Providers.Regulator.Mark(bodyJSON.Username, false)
```

Comment:
why not handle the error here?

Reply:
Addressed in https://github.com/authelia/authelia/pull/1511/commits/15646c4bedd408da2fe42ddef2c07318c3a54c25.